ui/webui/.stylelintrc.json

```diff
@@ -35,4 +35,4 @@
     "selector-class-pattern": null,
     "selector-id-pattern": null
```

Comment:
No big deal but this was introduced in the previous commit - so it should be fixed there.

Reply:
Done.